packages/beacon-node/src/execution/engine/http.ts

```diff
@@ -142,6 +157,11 @@ export class ExecutionEngineHttp implements IExecutionEngine {
     this.rpc.emitter.on(JsonRpcHttpClientEvent.RESPONSE, () => {
       this.updateEngineState(getExecutionEngineState({targetState: ExecutionEngineState.ONLINE, oldState: this.state}));
     });
+
+    // Initial engine state is ONLINE. Need to explicitly call `getClientVersion` once on startup
+    this.getClientVersion(getLodestarClientVersion(this.opts)).catch((e) => {
```

Comment:
calling this is the constructor still feels wrong to me, what happens if you start beacon node before EL client? this will log an error


maybe we should call it the first time we get a response from the EL if clientVersion == undefined